crates/verify/src/lib.rs

```diff
@@ -310,7 +316,6 @@ impl VerifyArgs {
                 output.artifact_ids().map(|(id, artifact)| (id, artifact.clone().into())),
             );
 
-            //TODO: lookup for zksync
```

Comment:
Does this happen now then?

Reply:
It tries to do it in `zk_resolve_context`. Did not really test it but it will compile the zk_project and do the lookup there. If that does not work for some reason we should move the TODO to that function as this function does not handle any zk stuff anymore

Reply:
I mean I left the todo there because I thought we might want this - if we now have this functionality then we can remove the todo, otherwise we need to add it back where it should be

Reply:
We do the lookup now so we should be good